examples/fl-tabular/federated.py

```diff
@@ -0,0 +1,186 @@
+from flwr.client import Client, ClientApp, NumPyClient
+from flwr.server import ServerApp, ServerConfig
+from flwr.server.strategy import FedAvg
+from flwr.simulation import run_simulation
+from flwr_datasets import FederatedDataset
+from flwr.common import ndarrays_to_parameters
+from sklearn.preprocessing import OneHotEncoder
+import pandas as pd
+import torch
+from torch.utils.data import TensorDataset, DataLoader
+import torch.nn as nn
+from collections import OrderedDict
+import torch.optim as optim
+from sklearn.preprocessing import OrdinalEncoder
+
+import numpy as np
+import random
+
+# Set seed for reproducibility
+SEED = 42
+random.seed(SEED)
+np.random.seed(SEED)
+torch.manual_seed(SEED)
+if torch.cuda.is_available():
+    torch.cuda.manual_seed(SEED)
```

Comment:
has the example been tested in GPU? My guess is that the way `initialize_model` has been designed it won't work since it's not sent to device.

Reply:
maybe we don't need CUDA for this small dataset/example?